src/autorest-core/lib/configuration.ts

```diff
@@ -232,18 +233,30 @@ export class ConfigurationView {
       // update source locations to point to loaded Swagger
       if (m.Source) {
         const blameSources = m.Source.map(s => {
-          try {
-            const blameTree = this.DataStore.Blame(s.document, s.Position);
-            const result = [...blameTree.BlameInputs()];
-            if (result.length > 0) {
-              return result.map(r => <SourceLocation>{ document: r.source, Position: Object.assign(TryDecodeEnhancedPositionFromName(r.name) || {}, { line: r.line, column: r.column }) });
+          let posClone = Clone(s.Position);
+          while (true) {
```

Comment:
This may be why you're regeneration doesn't terminate, if `result` is empty this loops forever.